node/nodeTesting.go

```diff
@@ -215,6 +215,56 @@ func (n *Node) GenerateAndSendBulkTransactionsOneByOne(receiverHex string, value
 	return nil
 }
 
+func (n *Node) SendBulkTransactions(txs []*transaction.Transaction) error {
```

Comment:
Might move this file into node.go. nodeTesting.go is due to be deleted (in the near future)

Reply:
done